nanoFirmwareFlasher.Tool/Esp32Manager.cs

```diff
@@ -0,0 +1,198 @@
+﻿//
+// Copyright (c) .NET Foundation and Contributors
+// See LICENSE file in the project root for full license information.
+//
+
+using System;
+using System.Linq;
+using System.Threading.Tasks;
+
+namespace nanoFramework.Tools.FirmwareFlasher
+{
+    /// <summary>
+    /// Class to manage different operations specific to the ESP32 platform.
+    /// </summary>
+    public class Esp32Manager : IManager
+    {
+        private readonly Options _options;
+        private readonly VerbosityLevel _verbosityLevel;
+
+        public Esp32Manager(Options options, VerbosityLevel verbosityLevel)
+        {
+            if (options == null) throw new ArgumentNullException(nameof(options));
```

Comment:
Project code style calls for open and close brackets on IF.

Reply:
👍